.github/workflows/bump_torch_mlir.yml

```
@@ -0,0 +1,67 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Automatically bumps the torch-mlir submodule to match shark-infra/torch-mlir:main.
+# Manually triggered for now, but can be scheduled in the future.
+# This typically takes ~90s to run.
+name: Bump submodule - shark-infra/torch-mlir
```

Let's drop the `shark-infra` part of this. We might move the forks that IREE uses (back) into `iree-org` and I don't see much reason to be this specific.
```suggestion
name: Bump submodule - torch-mlir
```